doc/run/repl.md

```diff
@@ -9,8 +9,8 @@ In addition to building SQLFlow into a gRPC server, accessed via Jupyter Noteboo
 The SQLFlow Docker image contains the REPL command-line program.  We can run an example session by typing the following command on MacOS.  If you run Docker on Linux, please change `host.docker.internal:3306` to `localhost:3306`.
 
 ```
-docker run -it --rm --net=host sqlflow/sqlflow repl \
---datasource="mysql://root:root@tcp(host.docker.internal:3306)/?maxAllowedPacket=0"
+docker run -it --rm --net=host sqlflow/sqlflow bash -c 'chmod +x start.sh; ./start.sh populate-example-dataset-mysql-local; \
```

Comment:
I think we can do chmod + x to the start.sh in the Git repo, so we don't need to chmod +x at the runtime; am I correct?